linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -51,6 +51,8 @@
 #define DEFAULT_QDISC_MAX_SIZE 50
 #define DEFAULT_QDISC_ENABLE_SIZE 10
 
+#define BLOCKING_FLOW 1
```

Comment:
2) oh come on... I put it in the .c files so it won't contaminate elsewhere, because if you put it in a header where other files may include it. I'd rather pass "true /* blocking */" than adding this as a definition in a common file. It has to go at some point sooner rather than later and moving the definition outside the file will mean we might miss it.